one_fm/grd/doctype/work_permit/work_permit.json

```diff
@@ -131,7 +131,6 @@
   {
    "fieldname": "employee",
    "fieldtype": "Link",
-   "ignore_user_permissions": 1,
```

Comment:
`ignore_user_permissions` should be true

Reply:
I did not tamper with that, i guess it is cos, i have not pulled from staging since i started working on this feature......but, i thought 1 also means True

Reply:
1 is true, but `-` it shows that we have removed it

Reply:
Done..